openhtf/util/logs.py

```diff
@@ -254,27 +290,31 @@ def format(self, record):
 
 
 @functions.call_once
-def configure_cli_logging():
-  """Configure OpenHTF to log to the CLI based on verbosity arg."""
+def configure_logging():
```

Comment:
Note: renaming this function is a breaking change for any testers or customer wrappers around openhtf that called configure_cli_logging().

Probably worth the temporary pain (can't make an omelet without breaking some eggs), but I believe the function name was also changed by Joe's major refactor of logs a couple months ago.